### ISSUE TITLE
Feature Request: verified links

### ISSUE THREAD

**Issue by Ambrosiani** [Automattic/wordpress-activitypub#506]
### What

Implement a mastodon "verified link" pointing to the wordpress site / wordpress author page.

### Why

increase legitimacy of the blog/author accounts

### How

No idea :/

**Comment by jazmichaelking** [comment]
A Mastodon "verified link" is to verify an account on a given server by referencing an external, third-party, authoritative domain, and is specific to Mastodon as a platform, it is not an ActivityPub function.

. A Wordpress account is not a Mastodon account, although it is visible on Mastodon and other platforms. 

For a verified profile on Mastodon, that Wordpress account owner would need to create a second account on Mastodon, then link to it from their web site.

More likely - it seems to me - accounts created on wordpress-example.com are de facto verified, and could use the relationship object https://www.w3.org/TR/activitystreams-vocabulary/#connections (eg user@wordpress-example.com is a participant or participantIn https://vocab.org/relationship/#participant)

This would lead to an AP Collection of authors, and they would be "verified" by their domain

In other words if I have an author account on wordpress-example.com, I am immediately verified on that domain, and federating clients should be able to discern that relationship using a standard object (and then determine if / how they might render that relationship in account summaries / profile cards etc)
